src/components/UI/Form/Calendar/Calendar.tsx

```diff
@@ -42,7 +42,6 @@ export const Calendar: React.SFC<CalendarProps> = ({
     <MuiPickersUtilsProvider utils={DateFnsUtils}>
       <Grid className={styles.Calendar}>
         <KeyboardDatePicker
-          error={hasError ? errorText : ''}
```

Comment:
This is needed to show the border as red on an error in the Calendar component.
It can be changed to `error={hasError}`
![image](https://user-images.githubusercontent.com/32592458/124547104-3fd78580-de49-11eb-96ed-19e07cbb3410.png)

Reply:
error is not actually the property of KeyboardDatePicker hence I had removed it. I am reverting this for now till we find a better solution.